examples/go-client/Dockerfile

```diff
@@ -9,4 +9,4 @@ RUN go get -u github.com/twilio/twilio-go@main
 RUN go get -u github.com/twilio/terraform-provider-twilio@main
 
 # pipefail prevents errors in a pipeline from being masked.
-CMD ["/bin/bash", "-c", "set -o pipefail && go test ./... -coverprofile /local/coverage.out -json | tee /local/test-report.out"]
+CMD ["/local/wait-for-prism.sh", "/bin/bash", "-c", "set -o pipefail && go test ./... -coverprofile /local/coverage.out -json | tee /local/test-report.out"]
```

Comment:
Since we have multiple services that will be dependent on the prism service being up and running, this script will need to exist in the context for each service. Instead can we replace this with a simple sleep:
```
CMD ["/bin/bash", "-c", "set -o pipefail && sleep 10 && go test ./... -coverprofile /local/coverage.out -json | tee /local/test-report.out"]
```

We've done something similar elsewhere which has not shown to have issues: https://github.com/sendgrid/sendgrid-oai/blob/64e32a322832ffea63afcd026e4e6557a04880ba/prism/docker-compose.yml#L8